financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/AccountItem.kt

```diff
@@ -48,103 +62,108 @@ internal fun AccountItem(
         modifier = Modifier
             .fillMaxWidth()
             .clip(shape)
             .border(
                 width = if (selected) 2.dp else 1.dp,
                 color = when {
-                    selected -> FinancialConnectionsTheme.colors.textBrand
-                    else -> FinancialConnectionsTheme.colors.borderDefault
+                    selected -> v3Colors.borderBrand
+                    else -> v3Colors.border
                 },
                 shape = shape
             )
-            .clickableSingle(enabled = selectable) { onAccountClicked(account) }
-            .padding(vertical = verticalPadding, horizontal = 16.dp)
+            .clickableSingle(enabled = selectable) {
+                if (SDK_INT >= M) view.performHapticFeedback(CONTEXT_CLICK)
+                onAccountClicked(account)
+            }
+            .alpha(if (selectable) SELECTABLE_ALPHA else UNSELECTABLE_ALPHA)
+            .padding(16.dp)
     ) {
         Row(
             horizontalArrangement = Arrangement.Start,
             verticalAlignment = Alignment.CenterVertically
         ) {
-            selectorContent()
-            Spacer(modifier = Modifier.size(16.dp))
+            networkedAccount?.icon?.default?.let {
+                InstitutionIcon(institutionIcon = it)
+                Spacer(modifier = Modifier.size(12.dp))
+            }
             Column(
-                Modifier.weight(ACCOUNT_COLUMN_WEIGHT)
+                Modifier.weight(1f)
             ) {
                 Text(
-                    text = title,
+                    text = account.name,
                     overflow = TextOverflow.Ellipsis,
                     maxLines = 1,
-                    color = if (selectable) {
-                        FinancialConnectionsTheme.colors.textPrimary
-                    } else {
-                        FinancialConnectionsTheme.colors.textDisabled
-                    },
-                    style = FinancialConnectionsTheme.typography.bodyEmphasized
-                )
-                subtitle?.let {
-                    Spacer(modifier = Modifier.size(4.dp))
-                    MiddleEllipsisText(
-                        text = it,
-                        color = FinancialConnectionsTheme.colors.textDisabled,
-                        style = FinancialConnectionsTheme.typography.captionTight
-                    )
-                }
-            }
-            networkedAccount?.icon?.default?.let {
-                StripeImage(
-                    url = it,
-                    imageLoader = LocalImageLoader.current,
-                    contentDescription = null,
-                    modifier = Modifier
-                        .size(16.dp)
+                    color = v3Colors.textDefault,
+                    style = v3Typography.labelLargeEmphasized
                 )
+                AccountSubtitle(selectable, account, networkedAccount)
             }
+            Spacer(modifier = Modifier.size(12.dp))
```

Comment:
Nit: Can we achieve the same by removing the `Spacer` calls and using `horizontalArrangement = Arrangement.spacedBy(12.dp)`?

Reply:
good catch https://github.com/stripe/stripe-android/pull/7736/commits/549e008235cdcd3e8682bef56c73dde934c992f1